### PR TITLE
Add Azuro Waves developer guide

### PR DESCRIPTION
**Pull request by on47sky**
Plus, add notice to Azuro Score pages

### CODE REVIEW

File: pages/hub/apps/guides/azuro-score.mdx [hunk 1]
```diff
@@ -1,203 +1,10 @@
-# Get Azuro Score leaderboard
+import { Callout } from 'components'
 
-Follow this guide to fetch data required to show Azuro Score leaderboard:
-- [Total](#fetch-total-leaderboard)
-- [Total including specific bettor](#fetch-total-leaderboard-including-specific-bettor)
-- [Increment by specific period](#fetch-leaderboard-for-specific-period)
-- [Increment by specific period including specific bettor](#fetch-leaderboard-for-specific-period-including-specific-bettor)
+# Get Azuro Score leaderboard
```
Comment (Avvrik) [inline pages/hub/apps/guides/azuro-score.mdx:3]: Get Azuro Waves leaderboard

File: pages/hub/apps/guides/azuro-score.mdx [hunk 2]
```diff
@@ -1,203 +1,10 @@
...
 
 If you not familiar with base concept about Azuro Score, read it [here](/concepts/azuro-score/overview).
```
Comment (Avvrik) [inline pages/hub/apps/guides/azuro-score.mdx:5]: better remove this as well

File: pages/hub/apps/guides/azuro-waves.mdx [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+  /** "2.100000", total user points without level multiplier */
+  points: `${number}`
+  /** "2.100000", final points with level multiplier ('boost') */
+  multipliedPoints: `${number}`
+}
+```
+
+## Base API URL
+
+For working with Azuro Waves data, we'll utilize the public API. The base URL is:
```
Comment (Avvrik) [inline pages/hub/apps/guides/azuro-waves.mdx:162]: To work with Azuro Waves data, we'll use the public API. ...

File: pages/hub/apps/guides/azuro-score.mdx (hunk 1 above)
Comment (on47sky) [reply pages/hub/apps/guides/azuro-score.mdx:3]: it's a page about Score

File: pages/hub/apps/guides/azuro-score.mdx (hunk 2 above)
Comment (on47sky) [reply pages/hub/apps/guides/azuro-score.mdx]: https://github.com/Azuro-protocol/gem-docs/pull/199/commits/6712abdfb5d5239837b7ffb1b11523d775b8c66d

File: pages/hub/apps/guides/azuro-waves.mdx (hunk 3 above)
Comment (on47sky) [reply pages/hub/apps/guides/azuro-waves.mdx]: https://github.com/Azuro-protocol/gem-docs/pull/199/commits/6712abdfb5d5239837b7ffb1b11523d775b8c66d

**Review by Avvrik — APPROVED** [review]

**Review by vaivanov — APPROVED** [review]

**Pull request merged**